client/src/Components/Pages/UserProfile/UserPoints.js

```diff
@@ -0,0 +1,65 @@
+import React, { Component } from "react";
+
+// common profile styles
+import {
+  BorderedSubSectionWrapper,
+  SubSectionTitleParagraph,
+} from "./UserProfile.style";
+
+import PopoverComponent from "../../Common/Popover";
+
+// individual styles
+import {
+  Wrapper,
+  ProgressSection,
+  Tracker,
+  YourProgressDiv,
+  ProgressTriangle,
+  ZeroNum,
+  ProgressNum,
+  CompetitionContainer,
+  ExtendedTracker,
+} from "./UserPoints.style";
+
+export default class UserPoints extends Component {
+  render() {
+    const { points } = this.props;
+    if (!points) return <div>Error loading your points...</div>;
```

Comment:
You should handle the case which `points = 0`
I got an empty page when I have 0 points
![image](https://user-images.githubusercontent.com/34629478/70697961-2dc64d00-1ccf-11ea-8dab-4104fc6ac21f.png)

Reply:
also could you update the blue text when 0 reviews been given
![Reviews you've given (none)](https://user-images.githubusercontent.com/34629478/70699345-a201f000-1cd1-11ea-967a-1dfaa4f57637.png)